radixdlt/src/test/java/com/radixdlt/sanitytestsuite/SanityTestSuiteTestLoader.java

```diff
@@ -0,0 +1,69 @@
+package com.radixdlt.sanitytestsuite;
+
+import com.google.common.hash.HashCode;
+import com.google.common.io.Files;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import com.google.gson.stream.JsonReader;
+import com.radixdlt.consensus.Sha256Hasher;
+import com.radixdlt.crypto.Hasher;
+import com.radixdlt.sanitytestsuite.model.SanityTestSuiteRoot;
+import com.radixdlt.utils.JSONFormatter;
+import org.json.JSONObject;
+
+import java.io.File;
+import java.io.FileReader;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.Assert.assertEquals;
+
+// CHECKSTYLE:OFF checkstyle:VisibilityModifier
```

Comment:
Please don't turn checkstyle checks off.  Better would be to not have public static members.